### ISSUE TITLE
Duplicate findings to Slack

### ISSUE THREAD

**Issue by pakeerubasha-mekala** [oli-pr/AWS-SecurityHubNotifications#1]
Hi,
 I am stilling getting duplicate notifications in slack. Is there any update?

**Comment by oli-pr** [comment]
Did the Stack successfully deploy?

This hasn't been reported before. Can you see the SentToSlack=True data tagged to your findings when you view them in SecurityHub?

**Comment by pakeerubasha-mekala** [comment]
Yeah lambda function deployed successfully. but I couldn't see SentToSlack=True tagging to findings in security hub>
